notebooks/fig1b.ipynb

```diff
@@ -457,7 +849,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 2",
+   "display_name": "OMERO Python 2",
```

Comment:
:+1: I realized these were being removed sometimes after testing as well. We may want to go to "Python 2" in production so that this time of reversion doesn't creep in. cc: @manics